### ISSUE TITLE
FR: Power recovery setting

### ISSUE THREAD

**Issue by Knud3** [bigtreetech/PandaPWR#11]
Currently, attached printer automatically power back on after a power outage, but providing a setting to remain powered off would give users greater control. This change accommodates those who prefer manual restart after outages, optimizing energy use and aligning with individual preferences. Adding this option enhances user satisfaction by offering more personalized power management.

**Comment by Rowan2233** [comment]
Thank you for this feedback bro, we will take this as next firmware consideration.

**Comment by Shadowtrance** [comment]
Second this. It should remember the last power state. Like PC's do with their power after AC loss setting in the bios.
Had a few power outages lately that have come back in the middle of the night and I'm woken up by my printer blasting light in my room (the p1s screen + ams leds are pretty dang bright in a dark room!)

So either remembering the last state or even a better a setting like a pc bios would be great.

**Comment by gunsha** [comment]
yes, this would be a really usefull feature to have. been having some power outages in the night lately and wake up to the printer turned on
